recipes/microprofile/all/conanfile.py

```diff
@@ -0,0 +1,218 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class MicroprofileConan(ConanFile):
+    name = "microprofile"
+    license = "Unlicense"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jonasmr/microprofile"
+    description = "Microprofile is a embeddable profiler in a few files, written in C++"
+    topics = ("profiler", "embedded", "timer")
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "microprofile_enabled": [True, False],
+        "use_miniz": [True, False],
```

Comment:
The recipe looks great!
My only remark is that we prepend options named after requirement with `with_`.
So it would be great if you would rename this option to `with_miniz`.